src/App.tsx

```diff
@@ -1,61 +1,132 @@
+import React, { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './types/userType';
+import { Todo } from './types/todoType';
+import { TodoList } from './components/TodoList/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export function getUser(userId:number):User | null {
+  const foundUser = usersFromServer.find(user => userId === user.id);
+
+  return foundUser || null;
+}
+
+export const fullTodoInfo:Todo[] = todosFromServer.map(todo => ({
```

Comment:
```suggestion
function getUserById(userId:number):User | null {
  const foundUser = usersFromServer.find(user => userId === user.id);

  return foundUser || null;
}

const fullTodoInfo:Todo[] = todosFromServer.map(todo => ({
```
Do you really need to export these variables?